blog/_posts/2021-10-06-manual-week.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "ROOT Manual's week"
+layout: archive
+author: Olivier Couet
+---
+
+During one full week all the ROOT team was involved in a big update of the
+_[Manual]({{ '/manual' | relative_url }})_ and _[Reference Guide](https://root.cern/doc/master/index.html)_.
+
+The ROOT documentation needed a big update.
+It was spread over three different main manuals:
+
+  - The _[Reference Guide](https://root.cern/doc/master/index.html)_
+  - The _[Manual]({{ '/manual' | relative_url }})_
+  - The _Old ROOT User’s Guide_
+
+The _[Reference Guide](https://root.cern/doc/master/index.html)_ and the
+_[Manual]({{ '/manual' | relative_url }})_ are the current valid sources of documentation.
+_[The Manual]({{ '/manual' | relative_url }})_ acts as a "User's Guide" helping users to find their way into the huge amount of documentation
+provided in the _[Reference Guide](https://root.cern/doc/master/index.html)_.
+The _Old User's Guide_ is outdated and not updated but, nevertheless it contains some valuable
+information we do not want to lose.
+
+The first task was to make sure the Manual’s table of content was complete: by
+groups of experts, we updated the existing chapters ([Histograms]({{ '/manual/Histograms' | relative_url }}),
+[Graphs]({{ '/manual/graphs' | relative_url }}), [Trees]({{ '/manual/trees' | relative_url }}) ...)
+and create the new needed ones ([JSROOT]({{ '/manual/jsroot' | relative_url }}), [
+ROOT I/O]({{ '/manual/root_io' | relative_url }})...) which also
+led to updates in the _[Reference Guide](https://root.cern/doc/master/index.html)_.
+
+Another task was be to move all the valuable information now in the _Old User’s
+Guide_ to the relevant places: outdated parts were removed or
+updated in the _[Manual]({{ '/manual' | relative_url }})_ or in the
+_[Reference Guide](https://root.cern/doc/master/index.html)_. Finally we will drop
+completely the _Old User’s Guide_ and have an accurate, minimal but sufficiently complete
+_[Manual]({{ '/manual' | relative_url }})_.
+
+The new structure _[Manual]({{ '/manual' | relative_url }})_/_[Reference Guide](https://root.cern/doc/master/index.html)_
+will work in a more complementary, simpler and proactive way for updates either
+via Pull requests (on [web site repository](https://github.com/root-project/web/pulls) or
+[ROOT repository](https://github.com/root-project/root)) or Github issues
+(on [web site repository](https://github.com/root-project/web/pulls) or
+[ROOT repository](https://github.com/root-project/root/pulls)).
```

Comment:
Maybe rephrase that to get our readers to interact? I think that was your plan? I.e. maybe:

This new manual allows *you* to contribute - for instance by letting us know when something is hard to understand (by opening an [issue](https://github.com/root-project/root/issues/new or whatever the link is)) or even by fixing it yourself: see the GitHub octocat at the bottom of each page!

And then conclude with something "warm":

We hope you enjoy the new manual, and that it's useful for today's new grad students!